cmd/backup.go

```diff
@@ -30,6 +31,11 @@ func runBackupCommand(command *cobra.Command, cmdName string) error {
 		ctx, store = trace.TracerStartSpan(ctx)
 		defer trace.TracerFinishSpan(ctx, store)
 	}
+	if cfg.IgnoreStats {
+		// Do not run stat worker in BR.
+		session.DisableStats4Test()
```

Comment:
What about restore?

Reply:
It's not easy to distinguish whether we should `DisableStats4Test` in restoration. I had two ideas
1.  Read backupmeta and check all tables whether stats != nil. but we should do it carefully, because read backupmeta is in `task.runRestore` and we cannot run `DisableStats4Test` otherwise it will influence BR via SQL.
2. Change kvproto to record the status of `--ignore-stats`. and this will involve other repo. 
So both of them will change a lot. we need to confirm the cost of `DisableStats4Test` is worth for above changes.

Reply:
[LoadStatsFromJson](https://github.com/pingcap/tidb/blob/76054200b6340f064e25046038d1ac1fdbe7e4ff/statistics/handle/dump.go#L198) only execute SQL to storage. so we can disable stats loop directly.